api/client.go

```diff
@@ -62,9 +62,9 @@ func (c *Client) Mutate(mutation interface{}, variables map[string]interface{})
 	return graphqlErr
 }
 
-func (c *Client) httpGET(path string) (*http.Response, error) {
+func (c *Client) HttpGET(path string) (*http.Response, error) {
```

Comment:
HttpGET, HttpPOST, HttpPUT, and HttpDELETE are all essentially the same. We could just have one method instead and optionally have these helpers around it with these same names?

Reply:
Sure. I've pushed a commit that merges them together to a single method.